packages/suite-web-landing/src/Main.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { IntlProvider } from 'react-intl';
+import { TranslationModeContext } from '../components/Translation';
+import { resolveStaticPath } from '@suite-utils/build';
+import Metadata from '@suite-components/Metadata';
+import { URLS } from '@suite-constants';
+import enLocale from '@trezor/suite-data/files/translations/en.json';
+import { TranslationModeTrigger } from '../components/LandingPage';
+import ThemeProvider from '@suite-support/ThemeProvider';
+import { Provider as ReduxProvider } from 'react-redux';
+import { Router as RouterProvider } from 'react-router-dom';
+import GlobalStyleProvider from '@suite-support/styles/GlobalStyleProvider';
+import AppRouter from './support/Router';
+import { store } from '@suite/reducers/store';
+import history from '@suite/support/history';
+
+const Main = () => {
+    const [translationMode, setTranslationMode] = useState(false);
+
+    return (
+        <>
+            <ReduxProvider store={store}>
+                <ThemeProvider>
+                    <RouterProvider history={history}>
+                        <GlobalStyleProvider />
```

Comment:
This component adds not only the old global styles but also `animations` and `notifications` styles which we do not need here. I would add some conditions there.